src/frontend-samples/read-settings-sample/ExternalSettingsUI.tsx

```diff
@@ -0,0 +1,169 @@
+/*---------------------------------------------------------------------------------------------
```

Comment:
This file name is inconsistent.  If the sample is called ReadSettingsSample then this file should be called ReadSettingApp.tsx.  But since there is code to also write settings maybe a different name is better.  But it should be consistent.

Reply:
Renamed file and class to be consistent